synapse/cortex.py

```diff
@@ -4646,6 +4651,22 @@ async def _onAddCronJob(self, cdef):
 
         return cdef
 
+    async def moveCronJob(self, useriden, croniden, viewiden):
+        view = self._viewFromOpts({'view': viewiden, 'user': useriden})
+
+        appt = self.agenda.appts.get(croniden)
+        if appt is None:
+            raise s_exc.NoSuchIden(iden=croniden)
+
+        if appt.view == view.iden:
+            return view.iden
```

Comment:
this should return None or the onMoveCronJob should return viewiden to make the return value behavior consistant

Reply:
:+1: